src/components/reimbursement-table/collapsible-row.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import * as React from "react";
+import { CaretSortIcon } from "@radix-ui/react-icons";
+import { Button } from "@/components/ui/button";
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from "@/components/ui/collapsible";
+
+interface MyComponentProps {
+  children: React.ReactNode;
+}
+
+const CollapsibleRowWrapper: React.FC<MyComponentProps> = ({ children }) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div>
+      <div
+        onClick={() => setIsOpen(!isOpen)}
+        style={{ cursor: "pointer", borderBottom: "1px solid #ccc" }}
```

Comment:
if possible, could you try to use tailwind classes instead of raw CSS styles to keep things consistent?